src/com/oltpbenchmark/benchmarks/tpcc/procedures/NewOrder.java

```diff
@@ -108,26 +117,76 @@
   private PreparedStatement stmtUpdateStock = null;
   private PreparedStatement stmtInsertOrderLine = null;
 
-  public ResultSet run(Connection conn, Random gen,
-    int terminalWarehouseID, int numWarehouses,
-    int terminalDistrictLowerID, int terminalDistrictUpperID,
-      TPCCWorker w) throws SQLException {
+  public NewOrder() {
+    stmtGetItemSQLArr = new SQLStmt[15];
+    stmtGetStockSQLArr = new SQLStmt[15];
+    stmtInsertOrderLineSQLArr = new SQLStmt[11];
+
+    // We create 15 statements for selecting rows from the `ITEM` table with varying number of ITEM
+    // ids.  Each string looks like:
+    // SELECT I_ID, I_PRICE, I_NAME , I_DATA
+    // FROM ITEM
+    // WHERE I_ID in (?, ? ..);
+    StringBuilder sb = new StringBuilder();
+    sb.append(String.format("SELECT I_ID, I_PRICE, I_NAME , I_DATA FROM %s WHERE I_ID in (",
```

Comment:
nit: for consistency capitalize `in` -> `IN`.

Reply:
Done